pytest.ini

```diff
@@ -1,7 +1,7 @@
 [pytest]
-addopts = -v -ra --cov
-python_paths = src/
+addopts = -v -ra --runslow
```

Comment:
Why did you get rid of the coverage report (`--cov` flag)?

Reply:
Ah sorry, totally forgot to add it back

Reply:
@dav3r resolved with [80f6d39](https://github.com/cisagov/domain-manager-api/pull/553/commits/80f6d399a0fb66ab207fa649406f678344eb5149)